pkg/gits/bitbucket_cloud.go

```diff
@@ -825,13 +820,40 @@ func (b *BitbucketCloudProvider) CreateIssue(owner string, repo string, issue *G
 }
 
 func (b *BitbucketCloudProvider) AddPRComment(pr *GitPullRequest, comment string) error {
-	log.Warn("Bitbucket Cloud doesn't support adding PR comments via the REST API")
-	return nil
+
+	rawComment := bitbucket.PullrequestComment{
+		Content: &bitbucket.IssueContent{
+			Raw: comment,
+		},
+	}
+
+	_, _, err := b.Client.PullrequestsApi.RepositoriesUsernameRepoSlugPullrequestsPullRequestIdCommentsPost(
+		b.Context,
+		pr.Owner,
+		pr.Repo,
+		int32(*pr.Number),
```

Comment:
Is it worth checking if this pointer is not nil before the conversion?

Reply:
Yeah that's a good point. I'll add a sanity check.